python/dgl/batch.py

```diff
@@ -2,7 +2,7 @@
 from collections.abc import Mapping
 
 from . import backend as F
-from .base import ALL, is_all, DGLError, dgl_warning, NID, EID
```

Comment:
Why did you remove `dgl_warning`? Is this not used?

Reply:
yes, it is not used after remove deprecate function.